pkg/destroy/gcp/gcp.go

```diff
@@ -118,6 +132,11 @@ func (o *ClusterUninstaller) Run() error {
 		return errors.Wrap(err, "failed to destroy cluster")
 	}
 
+	raw, err := json.Marshal(o.pendingItemTracker.removedQuota)
```

Comment:
There does not seem to be any way to pass options down from the top level to any individual destroyer, and I did not want to mess with those method signatures since they're pretty high-level. Open to ideas about how we can expose this output.

Reply:
In general, this seems like a fine approach to handle the serialization/publishing here, but maybe I'm just not following: if you could pass options down what would you do differently/what would that allow?

> Open to ideas about how we can expose this output.

Other than the current approach of writing to the install log, we could, of course, write to a new file. Brainstorming here, but we could consider adding this data to `metadata.json`. 

The [expressed purpose](https://github.com/openshift/installer/blob/master/pkg/asset/cluster/metadata.go#L41) of `metadata.json` is `// Metadata contains information needed to destroy clusters.` so this *may* be a bit of a stretch., but I don't think this is too far afield for consideration  I would just want to double check that metadata.json is left over after a successful destroy.